subtitles.go

```diff
@@ -392,6 +392,24 @@ type Metadata struct {
 	STLTranslatorName                                   string
 	Title                                               string
 	TTMLCopyright                                       string
+	WebVTTXTimestampMap                                 *TimestampMap
+}
+
+// TimestampMap is a structure for storing timestamps for WEBVTT's
+// X-TIMESTAMP-MAP feature commonly used for syncing cue times with
+// MPEG-TS streams.
+type TimestampMap struct {
```

Comment:
Could you rename this to `WebVTTTimestampMap` and move it to `webvtt.go` above `func parseWebVTTTimestampMap()`?